libpod/runtime.go

```diff
@@ -131,8 +131,9 @@ func NewRuntime(ctx context.Context, options ...RuntimeOption) (runtime *Runtime
 	if err != nil {
 		return nil, err
 	}
+	runtime, err = newRuntimeFromConfig(ctx, conf, options...)
```

Comment:
Is this right? We're making the runtime before we finalize the config, looks like?

Reply:
We need to take the input in from the user.  Currently if the user overrides the default cgroupfs, we report an error saying that the --cgroup-manager needs to be cgroupfs even if the user specified --cgroup-manager=cgroupfs.

This code is just verifying that the options that the runtime will use is correct.